### ISSUE TITLE
Endpoint must return the proper HTTP code

### ISSUE THREAD

**Issue by faelperetta** [vrnsky/recipeper-server#6]
Improve the controllers to return the proper HTTP code

For example, you can return HTTP code 201 when you create a new resource in a HTTP Post.

I think that currently the RecipeController always return HTTP Code 200, right?

**Comment by vrnsky** [comment]
@faelperetta 
When this method calls maybe thrown expceptio - RecipeNotFoundException - which already return http code.
``` 
     @RequestMapping(value = "/{id}", method = RequestMethod.GET)
    @ResponseBody
    public Recipe getById(@PathVariable Long id) {
        return recipeService.read(id);
    }
```
But other methods should return HTTP code 200 as you mentioned

**Comment by vrnsky** [comment]
Referenced ab0aaf1

**Comment by vrnsky** [comment]
Closed via #7
